src/components/organisms/tabs/TalksTab/index.jsx

```diff
@@ -0,0 +1,212 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> React Router bindings to DOM
+import { withRouter } from "react-router-dom";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBRow,
+  MDBCol,
+  MDBIcon,
+  MDBBtn,
+  MDBCard,
+  MDBCardHeader,
+  MDBCardBody,
+  MDBCardFooter,
+} from "mdbreact";
+
+//> CSS
+import "./talkstab.scss";
+//> Modules
+import { UploadModal } from "../../../molecules/modals";
+//#endregion
+
+//#region > Components
+class Talks extends React.Component {
+  state = {
+    showUpload: false,
+    loading: true,
+  };
+
+  handleUploadClose = () => {
+    if (this.state.showUpload) {
+      this.setState({
+        showUpload: false,
+      });
+    }
+  };
+
+  updateIframe = (talk) => {
+    let iframe;
+
+    if (talk.interval.loaded === false) {
+      if (document.getElementById(talk.uid)) {
+        iframe = document.getElementById(talk.uid);
+        iframe.src = talk.displayUrl;
+      }
+    }
+  };
+
+  render() {
+    const { globalState, globalFunctions } = this.props;
+    const talkList = globalState.fetchedUser.platformData.talks;
```

Comment:
Is this stable? I would improve it.
```suggestion
    const talkList = globalState?.fetchedUser?.platformData?.talks;
```